### DISCUSSION TITLE
Does not capture audio from iPad ARM Apps

### DISCUSSION THREAD

**Discussion by kevinbkt10** [ExistentialAudio/BlackHole#602] [Q&A]
I'm trying to use OBS to capture the sound of an App Store game on M1 called Stumble Guys, but its internal sound is not captured by Black Hole, youtube and other apps captures normal, but this game doesn't... There was an hour that I could capture, but only he could not make the audio come out of the speakers, even using several attempts to configure aggregate device and multiple outputs.

Either it's one or the other, I can't hear the sound when I capture it or I don't capture the sound when I hear it, it happens only in this APP, browser I can do both together quietly. It's not OBS's problem because I'm seeing the capture test in inputs in the audio settings.

**Comment by YYAppleFan** [comment]
Same problem,but in the game Arknights.

**Comment by YYAppleFan** [comment]
Tested. It's half fixed if I use BlackHole as the ONLY output device. Using Multi-output doesn't work still.

**Comment by YYAppleFan** [comment]
One more thing, OBS cannot capture the audio even when I set BlackHole as the only output device.

**Comment by devinroth** [comment]
Some apps don't support aggregate devices. Sorry.
